sections/product-recommendations.liquid

```diff
@@ -131,6 +133,34 @@
       "label": "t:sections.product-recommendations.settings.show_rating.label",
       "info": "t:sections.product-recommendations.settings.show_rating.info"
     },
+    {
+      "type": "select",
+      "id": "color_scheme",
+      "options": [
+        {
+          "value": "accent-1",
+          "label": "t:sections.all.colors.accent_1.label"
+        },
+        {
+          "value": "accent-2",
+          "label": "t:sections.all.colors.accent_2.label"
+        },
+        {
+          "value": "background-1",
+          "label": "t:sections.all.colors.background_1.label"
+        },
+        {
+          "value": "background-2",
+          "label": "t:sections.all.colors.background_2.label"
+        },
+        {
+          "value": "inverse",
+          "label": "t:sections.all.colors.inverse.label"
+        }
+      ],
+      "default": "background-1",
+      "label": "t:sections.all.colors.label"
+    },
```

Comment:
I think it should be move above and not within product cards: ![](https://screenshot.click/22-01-51ncw-3vkr9.png)

Reply:
Was there UX guidance for the color settings add-on? I suspect we'll need help from @katycobb to either change the setting label for `Section color scheme` or place it under a header called as `Section` and have `Top padding`, `Bottom padding` live under the same header. cc @Oliviammarcello

Reply:
Good point!

Reply:
Yes @katycobb and I reviewed it together in figma. We opted to not include the word 'Section' in front of the colour scheme because it would get repetitive and we don't use 'section' for other settings that apply to the entire section in most areas. 

There are a few inconsistencies as Ludo pointed out (I will also review all tomorrow). All screens should match what is done on this [page.](https://www.figma.com/file/0bdfenAhtGXTiH93THYx0a/Colour?node-id=214%3A6728) Since the section padding was added before we updated Figma, to clarify, all colour scheme settings should be included above section padding.

Reply:
One thing - I noticed Slideshow as one of the sections, but that one's colour schemes are set block by block so there is no section setting for it that makes sense.

Reply:
Yup! Because the slideshow is full width we decided not to add it to the section level. Similar to Image banner.